server/eb/app.py

```diff
@@ -98,7 +98,11 @@ def _before_adding_routes(app, app_config):
         server_config = app_config.server_config
         # This hash should be in sync with the script within
         # `client/configuration/webpack/obsoleteHTMLTemplate.html`
-        obsolete_browser_script_hash = ["'sha256-wl4OlniJEAFM1/VNFG/LBDQyPRkO7P/5kfQWf3+fPWE='"]
+
+        # It is _very_ difficult to generate the correct hash manually,
+        # consider forcing CSP to fail on the local server by intercepting the response via Requestly
```

Comment:
Super useful comment! Should we add the PR description link here too, so people can find your "How To" steps :D? Thank you!

Reply:
Good idea!